tests/httplib2test.py

```diff
@@ -7,6 +7,7 @@
 Requires Python 2.4 or later
 """
 
+from shotgun_api3.lib import six
```

Comment:
are these the official tests for httplib2 - why do we need to run them?

Reply:
As discussed offline, we'll remove them.

Reply:
The file is gone 👻